src/components/CustomButton/index.stories.tsx

```diff
@@ -50,16 +28,16 @@ const Template: Story<ButtonProps> = (args) => (
   >
     <Button {...args} />
     <Button {...args}>
-      <CloseIcon /> Custom Button
+      <Icon.Close fill="currentColor" /> Custom Button
```

Comment:
Isn't currentColor the default here either way?

Reply:
No, the default color for this icon is `#6D6D6D`. Should we update all icons to have `currentColor` as the default value?

Reply:
oufff... umm we probably can't without breaking some UIs so we will need to align with the team befoer we do that.